database/network-database/scripts/loader.py

```diff
@@ -74,6 +79,11 @@ def LOAD_NETWORK():
             row_num += 1
     print('\\.')
 
-LOAD_SOURCES()
-LOAD_GENES()
-LOAD_NETWORK()
+# load_sources()
+Utils.load_sources(Constants.NETWORK_DATA_SOURCE, Constants.DATABASE_NAMESPACE)
+
+# load_genes()
+Utils.load_genes(Constants.GENE_SOURCE, Constants.DATABASE_NAMESPACE, False)
+
+# load_network()
+Utils.load_network(Constants.NETWORK_SOURCE, Constants.DATABASE_NAMESPACE, False)
```

Comment:
It’s typically an anti-pattern to have unnamed Boolean arguments because other code readers will not clearly know what is being set to true or false; let’s make this a named parameter

Name tbd pending inclusion of _utils_ module in the PR…